buildozer/__init__.py

```diff
@@ -476,23 +476,6 @@ def file_matches(self, patterns):
     def file_exists(self, *args):
         return exists(join(*args))
 
-    def file_rename(self, source, target, cwd=None):
```

Comment:
Seems we still have a reference to `file_rename` here:
https://github.com/kivy/buildozer/blob/b1f0e61ef30462df021f08ef1dc15bf10e40e8d6/buildozer/targets/ios.py#L313